### ISSUE TITLE
If the swipe-open attribute is set, the address bar never disappears in Chrome Mobile when scrolling down the page

### ISSUE THREAD

**Issue by mercmobily** [PolymerElements/app-layout#316]
If the drawer is defined as:

    <app-drawer swipe-open>

Then any page viewed in Chrome will have the address bar stuck to it.

Reproducing is simple: it happens with the current PSK. Just add `swipe-open` to the `app-drawer` and you will see it happen: scrolling down the content, the address bar won't disappear,

Taking `swipe-open` out, you will see that the browser's address bar will disappear when scrolling down.
Not sure about Safari, sorry.

**Comment by mercmobily** [comment]
Err... wooops. The problem seems to be with:

    <app-header-layout has-scrolling-region>

If `has-scrolling-region` is set, then the address bar in the browser always displays, even after scrolling down...

**Comment by keanulee** [comment]
@mercmobily That's one of the limitations if you use an element scroller (e.g. by using `has-scrolling-region` instead of the document scroller. For more details, see https://www.polymer-project.org/1.0/toolbox/app-layout#document-scroller-and-element-scrollers

**Comment by mercmobily** [comment]
I don't suppose I could ask you a favour and check that I got this 100% right?

I can see that has-scrolling-region has the following effect:

      :host > ::content > app-header {
        @apply(--layout-fixed-top);
        z-index: 1;
      }
      :host([has-scrolling-region]) {
        height: 100%;
      }
      :host([has-scrolling-region]) > ::content > app-header {
        position: absolute;
      }

Since the local DOM is:

<content id="header" select="app-header"></content>

    <div id="contentContainer">
      <content></content>
    </div>

Let me try to explain in plain English what I see here.
If `[has-scrolling-region]` is set:

* The host element (that is, the `app-header-layout`, containing both the header and the pages) has an height of 100%.

* The `app-header` element in the local DOM has absolute positioning. This means that it will not move and won't scroll up no matter what

* The container of the `<content>` tag in the light DOM (that is, anything apart from the the header) has `layout: fit`, which basically means `position:absolute; top,bottom,left,right:0`. Since the host (app-header-layout` is `position: relative`, the anchor element will be `app-header-layout`. The container ALSO has `overflow-y: auto;`, which basically means that the scrollbars will be displayed

So, the document itself will never scroll. The containing element will.

Now, two questions:

1) Is it worthwhile for me to write a tutorial about this?
2) Is there really no way to make it artificially scroll? Maybe by making the containing element a little bigger than 100%, and manually scrolling down? Or (maybe better) by getting the `app-header` to check if `content` is scrolled down a little, and if it is, disappear? (Basically, doing this programmatically)

Sorry about the noise.

**Comment by keanulee** [comment]
You're right - because the height essentially becomes 100%, the document itself never really scrolls. `app-header` may "scroll up", but that's because we use `transform` on the element. If you want the address bar to scroll away, either use document scrolling (which has its set of drawbacks, mainly you can only have one document scroller per page), or, like you mentioned, try to set the height larger than 100% (but that's outside the scope of app-header-layout; maybe you can just set `app-header-layout`'s height, but that might require some investigation).

**Comment by mercmobily** [comment]
Thank you! I will play with it. Making the height a tiny more than 100%
will probably work, but we might end up with the browser's scrollbar to
appear, which would be ugly. Will look into it, and if I am successful I
will provide a patch.
